cypress/support/pages/mp/sales/sales-orders-page.ts

```diff
@@ -13,8 +13,9 @@ export class SalesOrdersPage extends MpPage {
 
   find = (params: FindParams): Cypress.Chainable => {
     const searchSelector = this.repository.getSearchSelector();
-    cy.get(searchSelector).clear();
+    cy.get(searchSelector, { timeout: params.timeout ?? 5000 }).clear();
```

Comment:
Should we override the default timeout on this page for all tests or for a specific one?

According to the docs, `defaultCommandTimeout` is 4000ms.
https://docs.cypress.io/app/references/configuration#Timeouts

If it's a flicker for only 1 test, let's do something like this
```suggestion
    cy.get(searchSelector, { timeout: params.timeout ?? Cypress.config('defaultCommandTimeout') }).clear();
```